ui/pages/settings/networks-tab/networks-form/networks-form.tsx

```diff
@@ -0,0 +1,634 @@
+import log from 'loglevel';
```

Comment:
This converts the existing `networks-form.js` to typescript and greatly simplifies it, while adding our new features